### ISSUE TITLE
Space Bar does not work on IPadOS 13 when Full Keyboard Access is enabled

### ISSUE THREAD

**Issue by glang0** [holzschu/a-shell#45]
On my iPad (iPro 4th gen 12.9” with Magic Keyboard) the space bar is not received by the app. All other keys work fine, even [cmd-Space] to pull up iOS’s search bar when in the app. This seems to be because in Settings, one of the “Controls” that are enabled is “Activate”. Once full keyboard access is disabled everything works fine. 

I also tested this on LibTerm and there seemed to be no issues within that app. I would try and compare differences between both app’s ios_project repository, I just have little knowledge in Obj-C and Swift. 

Thanks again for making this project! A local vim install with a shell is something that is sorely needed on iOS.

**Comment by holzschu** [comment]
Now that one is going to be difficult for me to debug for the time being...
My favorite suspect for issues with the keyboard is the settings "use caps-lock as escape". I think it is deactivated by default, so it's probably not that.

**Comment by janngobble** [comment]
Just for the record? This is still an issue on 16.2... I know it's gonna be hard to debug. Just saying it's a problem still.
